### PR TITLE
[Finishes #187355055 #187355057 #187848506] seller-page

### PR DESCRIPTION
**Pull request by venuste-20002**
1: what does this PR do?

 one this  PR A seller should be able to view a single item and have a different view of all Items in seller's collection,
 A seller should be able to update an product item from their collection  and 
 A seller should be able to delete a specific item from their collection

2: Relevant Stories:

### https://www.pivotaltracker.com/story/show/187848506

3: Screenshots

### CODE REVIEW

File: src/Pages/sellerSingleProduct/sellerSingleProduct.tsx [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  const handlePrev = () => {
+    setActiveIndex((prevIndex) => (prevIndex === 0 ? images.length - 1 : prevIndex - 1));
+  };
+
+  const handleNext = () => {
+    setActiveIndex((prevIndex) => (prevIndex === images.length - 1 ? 0 : prevIndex + 1));
+  };
+
+  const handleDelete = (index: number) => {
+    if (window.confirm("Are you sure you want to delete this image?")) {
```
Comment (UmuhireJessie) [inline src/Pages/sellerSingleProduct/sellerSingleProduct.tsx]: Can you please use a modal instead of an alert

File: src/asios.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+//     const localData= localStorage.getItem('loginData');
+
+//      request.headers.authorization = `Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiIwMzJmMzVlNS00YzQxLTQ1MTItYTVlMy04YTZhZjljYWVjODEiLCJlbWFpbCI6InJhZGVlbi5hc2h0ZW5AZG9ja3N0b25lcy5jb20iLCJmaXJzdE5hbWUiOiJzdHJpbmciLCJsYXN0TmFtZSI6InN0cmluZyIsInBhc3N3b3JkTGFzdENoYW5nZWQiOiIyMDI0LTA3LTE2VDEyOjU3OjQwLjQ3OVoiLCJyb2xlIjoic2VsbGVyIiwiaXNWZXJpZmllZCI6dHJ1ZSwiaWF0IjoxNzIxMTM1MDg0LCJleHAiOjE3MjEyMjE0ODR9.qIXuH-TR89evgvqazx6ABSGwpMZ7v2BEmIO--3EfODo`;
+    
+
+//     return request;
+// },
+// (error) => Promise.reject(error)
+// )  
+
```
Comment (UmuhireJessie) [inline src/asios.tsx]: Remove these comments.

File: src/redux/store.ts [hunk 3]
```diff
@@ -13,6 +14,12 @@ import productReviewReducer from './slices/productSlice'
 import cartCheckoutReducer from "./slices/cartCheckoutSlice";
 import addressSlice from './slices/addressSlice';
 
+<<<<<<< HEAD
+=======
+// Combine all reducers into a single root reducerimport productReducer from './slices/productSlice';
+import deletedReducer from './slices/deleteSlice';
+
+>>>>>>> b05dd2c ([Finishes #187355055 #187355057 #187848506] seller-page)
```
Comment (UmuhireJessie) [inline src/redux/store.ts]: Please ensure that you have resolved conflicts well

File: src/redux/store.ts [hunk 4]
```diff
@@ -23,9 +30,14 @@ const rootReducer = combineReducers({
...
   cart: cartSlice,
   navbarSlice:navbarSlice,
+<<<<<<< HEAD
   cartCheckout: cartCheckoutReducer,
   singleItemOrder: singleItemOrderReducer,
   productReviews: productReviewReducer
+=======
+  product: productreducerform,
+  deleted: deletedReducer,
+>>>>>>> b05dd2c ([Finishes #187355055 #187355057 #187848506] seller-page)
```
Comment (UmuhireJessie) [inline src/redux/store.ts]: Please resolve conflicts satisfactorily.

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
Please check my comments

**Review by Celestin4 — APPROVED** [review]

**Review by Christian-Ishimwe — APPROVED** [review]

**Review by Christian-Ishimwe — APPROVED** [review]
Can you work on resolving the conflicts?

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
@venuste-20002 please work on:
1. Updating the page after login without refresh. The page is just blank
Screenshot 2024-07-22 at 20 04 13

2. Consider improving the design of this search box and the side bar. Consult Shema's work as a reference on the width of side bar
Screenshot 2024-07-22 at 20 08 04

**Review by lilika67 — APPROVED** [review]

**Pull request closed without merging**